src/components/Loader/person.tsx

```diff
@@ -0,0 +1,59 @@
+import { Link } from "react-router-dom"
+import { Person } from "../../types"
+import classNames from "classnames"
+
+type Props = {
+  person: Person
+  mothers: Person[]
+  fathers: Person[]
+  currentPath: string
+}
+
+
+export const People: React.FC<Props> = ({ person, fathers, mothers, currentPath }) => {
+  return (
+    <tr data-cy="person" key={person.slug} className={classNames({
+      'has-background-warning': currentPath === `/people/${person.slug}`
+    })} >
+      <td>
+        <Link
+          to={`/people/${person.slug}`}
+          className={classNames({'has-text-danger': person.sex === 'f'})}
+        >
+          {person.name}
+        </Link>
+      </td>
+
+      <td>{person.sex}</td>
+      <td>{person.born}</td>
+      <td>{person.died}</td>
+
+      <td>
+        {mothers.map(mother => mother.name).includes(person.motherName || '-') ? (
+          <Link
+            className="has-text-danger"
+            to={`/people/${mothers
+              .find(mother => mother.name === person.motherName)?.slug}`}
+          >
+            {person.motherName}
+          </Link>
+        ) : (
+          `${person.motherName || '-'}`
+        )}
+
+      </td>
+
+      <td>
+        {fathers.map(father => father.name).includes(person.fatherName || '-') ? (
+          <Link to={`/people/${fathers
+            .find(father => father.name === person.fatherName)?.slug}`}
+          >
+            {person.fatherName}
+          </Link>
+        ) : (
+          `${person.fatherName || '-'}`
```

Comment:
try to find the mother or father before jsx and just use it, you will simplify the logic and remove the includes method